server/service/IndividualServicesService.js

```diff
@@ -146,7 +146,7 @@ exports.bequeathYourDataAndDie = function (body, user, originator, xCorrelator,
       let applicationPort = body["new-application-port"];
 
       
-    let newReleaseUuids = await resolveHttpTcpAndOperationClient(FcportValue)
+    let newReleaseUuids =  await LogicalTerminationPointServiceOfUtility.resolveHttpTcpAndOperationClientUuidFromForwardingName()
```

Comment:
Kindly please let me know whether this method doesnt require an argument to get the newReleaseuuids?

Reply:
we are passing the forwarding name inside the resolveHttpTcpAndOperationClientUuidFromForwardingName method since AA EATL OL ALT use the same method and same Forwarding name so